src/WebJobs.Script.WebHost/App_Start/WebHostResolver.cs

```diff
@@ -267,6 +267,13 @@ private TInstance GetActiveInstance<TInstance>(WebHostSettings settings, ref TIn
 
         private void OnSpecializationTimerTick(object state)
         {
+            if (_settingsManager.IsZipDeployment)
+            {
+                // TEMP - when in Zip deploy mode, temporarily disabling timer based specialization
+                // until issue https://github.com/Azure/azure-functions-host/issues/2392 is fixed.
+                return;
+            }
```

Comment:
Before your previous change, I think we already had the first EnsureInitialized, but not the second. Just want to make sure that we're ok with not having the first at all.

Reply:
We want to skip all of this. This OnSpecializationTimer change came in 2 commits, the second being a bug fix adding the second call.

For this tactical fix, we're saying that when in Zip mode we're relying on the http stimulus to trigger specialization.